### DISCUSSION TITLE
Usage Of Arc's Service Worker

### DISCUSSION THREAD

**Discussion by gaurishhs** [quasarframework/quasar#11082] [CLI - PWA mode]
Hello, 
Im New To Using Service Workers in Quasar. 

How Can we Use Service Workers in Quasar? Like Custom Registering Them Because arc enforces users to use its own Service Worker Which is available at https://arc.io/arc-sw.js

**Comment by MilosPaunovic** [comment]
You can check official docs example:
https://quasar.dev/quasar-cli/developing-pwa/handling-service-worker
